e2e_tests/tests/per_provider/normal_tests/import_key.rs

```diff
@@ -131,18 +141,18 @@ fn create_and_import_rsa_key() -> Result<()> {
     Ok(())
 }
 
-#[cfg(feature = "cryptoauthlib-provider")]
+#[cfg(any(feature = "mbed-crypto-provider", feature = "cryptoauthlib-provider"))]
 #[test]
 fn create_and_import_ecc_key() -> Result<()> {
     let mut client = TestClient::new();
-    let key_name = String::from("create_and_import_key");
+    let key_name = String::from("create_and_import_ecc_key");
     if !client.is_operation_supported(Opcode::PsaImportKey) {
         return Ok(());
     }
 
     client.generate_ecc_key_pair_secpr1_ecdsa_sha256(key_name.clone())?;
     let status = client
-        .import_ecc_public_secp_r1_ecdsa_sha256_key(key_name, KEY_PAIR_ECC.to_vec())
+        .import_ecc_public_secp_r1_ecdsa_sha256_key(key_name, ECC_KEY_PAIR.to_vec())
```

Comment:
I guess it makes no difference here as we check if the key already exists (so that the code does not go as far as parsing it), but I think it should be `ECC_PUBLIC_KEY`.